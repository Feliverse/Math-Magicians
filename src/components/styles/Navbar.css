.navbar {
  padding: 0 1rem;
  display: flex;
  justify-content: space-between;
  align-items: center;
  height: 80px;
  background-color: #f8f5f1;
}

.navlinks {
  list-style: none;
  padding: 0;
  margin: 0;
  display: flex;
  justify-content: space-around;
  align-items: center;
}

a {
  padding-right: 1rem;
  color: #333;
  text-decoration: none;
}

h1 {
  font-size: 200%;
  background: radial-gradient(#ffa600, orange, yellow);
  -webkit-background-clip: text;
  color: transparent;
}

/* Active route */
.active {
  color: #22b436;
  text-decoration: underline;
  text-shadow: #22b4357a 0 0 16px;
  font-weight: bold;
}
