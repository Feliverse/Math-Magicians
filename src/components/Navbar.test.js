import { render } from '@testing-library/react';
import { BrowserRouter as Router } from 'react-router-dom';
import '@testing-library/jest-dom/extend-expect';
import Navbar from './Navbar';

test('renders Header component', () => {
  const { asFragment } = render(<Router><Navbar /></Router>);
  expect(asFragment()).toMatchSnapshot();
});
