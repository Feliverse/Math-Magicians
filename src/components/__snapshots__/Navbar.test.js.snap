// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders Header component 1`] = `
<DocumentFragment>
  <nav
    class="navbar"
  >
    <h1>
      Math Magicians
    </h1>
    <ul
      class="navlinks"
    >
      <li>
        <a
          aria-current="page"
          class="active"
          href="/"
        >
          Home
        </a>
      </li>
      <li>
        <a
          class=""
          href="/calculator"
        >
          Calculator
        </a>
      </li>
      <li>
        <a
          class=""
          href="/quote"
        >
          Quote
        </a>
      </li>
    </ul>
  </nav>
</DocumentFragment>
`;
